src/muqsit/vanillagenerator/generator/biomegrid/RiverMapLayer.php

```diff
@@ -5,12 +5,12 @@
 namespace muqsit\vanillagenerator\generator\biomegrid;
 
 use muqsit\vanillagenerator\generator\overworld\biome\BiomeIds;
-use pocketmine\world\biome\Biome;
+use pocketmine\data\bedrock\BiomeIds as PMBiomeIds;
 
 class RiverMapLayer extends MapLayer{
 
 	/** @var int[] */
-	private static $OCEANS = [Biome::OCEAN => 0, BiomeIds::DEEP_OCEAN => 0];
+	private static $OCEANS = [PMBiomeIds::OCEAN => 0, BiomeIds::DEEP_OCEAN => 0];
```

Comment:
A better solution would be mapping the biome ID constants in `muqsit\vanillagenerator\generator\overworld\biome\BiomeIds` to the constants in `pocketmine\data\bedrock\BiomeIds`. VanillaGenerator's BiomeIds has more biome ids than pocketmine\data's. Its too messy to use both the classes together in every file.

Reply:
It looks like the difference with Pocketmines is that they have MOUNTAINS as 3, which is EXTREME_HILLS in your id constants as well as SWAMP being SWAMPLANDS. You want me to add pocketmines constants to yours and stick with yours?

This is my change in BiomeIds.php

```suggestion
public const EXTREME_HILLS, MOUNTAINS = 3;
	public const FOREST = 4;
	public const TAIGA = 5;
	public const SWAMPLAND, SWAMP = 6;
```

Reply:
Hmm, sounds like a bug. I imported the biome ids that MCPC has, which probably differs from the ones MCPE uses.

Reply:
I'd suggest something like
```php
namespace muqsit\vanillagen...;

use pocketmine\data\bedrock\BiomeIds as VanillaBiomeIds;

final class BiomeIds{

	public const OCEAN = VanillaBiomeIds::OCEAN;
}
```